scripts/check_style.sh

```diff
@@ -4,7 +4,7 @@
 
 (
 REPO_ROOT="$(dirname "$0")"/..
-cd $REPO_ROOT
+cd $REPO_ROOT || exit 1
```

Comment:
Why not just `set -e`?

Reply:
Same as in the other PR. I didn't want to do it in such a big batch of changes to avoid introducing bugs.

Reply:
Done. This required ignoring the return values from `grep`.